src/_gettsim/interface.py

```diff
@@ -40,8 +39,6 @@
     get_path_for_group_by_id,
     merge_trees,
     partition_tree_by_reference_tree,
-    qualified_name_reducer,
-    qualified_name_splitter,
 )
 
 
```

Comment:
Just a note that we should eventually move to `targets` meaning `targets_tree_paths`. Pretty unnatural to be using a dict at the lowest level here! *(but of course the best we could think of before diving deep into `dags.tree`)*. Same for the input structure elsewhere.